misc/install-git-hooks.py

```diff
@@ -63,7 +63,7 @@ def main():
 
         if not os.path.exists(sh_path):
             sh_path = os.path.join(
-                os.path.dirname(git_path), "..", "..", "bin", "sh.exe"
```

Comment:
That's strange, why is it working for me then.

Reply:
Found out myself, it only works when run from git "bash", but not from "cmd.exe"